src/metabase/api/user.clj

```diff
@@ -320,5 +321,13 @@
       (email/send-new-user-email! user @api/*current-user* join-url)))
   {:success true})
 
+(api/defendpoint DELETE "/:id/subscriptions"
+  "Delete all Alert and DashboardSubscription subscriptions for a User. Only allowed for admins or for the current
+  user."
+  [id]
+  (check-self-or-superuser id)
+  (db/delete! PulseChannelRecipient :user_id id)
```

Comment:
does this invoke the logic to remove pulses and subscriptions after the last recipient is removed? Not sure if that's been merged yet or if that happens automatically on the post delete

Reply:
i read the tests more closely and see now. sorry :)